fingertip/main.py

```diff
@@ -46,7 +47,15 @@ def main():
     args = sys.argv[1:]
 
     if not args:
-        fingertip.util.log.error('no plugin specified')
+        info = """ Name:
+        fingertip
+ Usage:   
```

Comment:
1) Why is Usage not indented?
2) could be "fingertip: a tool to control VMs/containers"
3) usage messages sometimes use `sys.argv[0]` as a command to match how the user has invoked it

Also, would be cool to stay under 79 character line limit, but I'm not sure this is reasonable in this particular case.